README.md

```diff
@@ -28,33 +28,33 @@ Connect your Github code with your project management in Jira. A separate Jira s
 This app officially supports GitHub.com and Jira Cloud. Support for GitHub Enterprise and/or Jira server may be considered in the future.
 
 ### Installation from Atlassian
-1. Sign into your Jira Cloud account
-2. Open the left sidebar by clicking on **Personal Settings**, if the left side bar is not already open. From the left sidebar in Jira, select **Jira Settings** -> **Apps** -> **Find new Apps**. (If you're using an older version of Jira, you won't have a left sidebar. Instead, click the **Gear Icon** in the top-right corner and select **Settings**. From there, select **Manage add-ons** from the left sidebar.)
-3. Search for **GitHub for Jira** and Click **Install**
-![image](https://user-images.githubusercontent.com/13207348/46588299-08550800-ca68-11e8-8ed4-290533320ef4.png)
+1. Sign in to your Jira Cloud account.
+2. From the top navigation bar in Jira, select **Apps** -> **Find new Apps**. You can also click the **Gear Icon** in the top-right corner and select **Apps**. 
+3. Search for **GitHub for Jira** and Click **Get app**
+![image](./docs/images/install-app-atlassian.png)
 7. Click the **Get Started** button to connect your GitHub account.
 
-Next you will need to connect your GitHub organization to Jira, see the following steps.
+Next you will need to [connect your GitHub organization to Jira](#connecting-your-github-organization-to-jira).
 
 ### Installation from GitHub Marketplace
 1. Go to https://github.com/marketplace/jira-software-github
 2. Complete the (free) order for your GitHub Organization
 3. On the installation setting screen, choose which repositories you want to use with the Jira Integration and press **Save**:
 ![image](https://user-images.githubusercontent.com/13207348/46588321-4baf7680-ca68-11e8-872a-a6d48924d655.png)
-4. Once installation completes, you will be redirected to https://jira.github.com/github/setup. Enter the site name for your Jira instance here and click **Continue**
```

Comment:
This URL is broken, and users won't be redirected here anyways. To be honest, the flow of installing from GitHub is a bit of a mess right now, but tracked at [ARC-673]. cc @rachellerathbone

Reply:
I ended up making changes to this with the fix that got merged for the broken flow